packages/website/src/lib/constants.ts

```diff
@@ -30,3 +30,20 @@ export type VersionSlug = (typeof slugs)[number]
 export function isValidVersionSlug (maybeSlug: string | undefined): maybeSlug is (typeof slugs)[number] {
   return slugs.includes(maybeSlug as (typeof slugs)[number])
 }
+
+export function algoliaEnv () {
+  const ALGOLIA_APP_ID = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID
+  const ALGOLIA_API_KEY = process.env.NEXT_PUBLIC_ALGOLIA_API_KEY
```

Comment:
Just want to verify this is safe to add? I noticed that the backend uploader also uses an API key so I want to ensure that people cannot mess with the search index if this is made public.

Reply:
a readonly search api key is available. updated